### PR TITLE
Update snap binary names to snaptel/snapteld

### PR DESCRIPTION
**Pull request by mkleina**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -90,44 +90,44 @@ $ curl -sfLO https://raw.githubusercontent.com/intelsdi-x/snap-plugin-collector-
 Ensure to provide your Keystone instance address and credentials.
 
 Ensure [Snap daemon is running](https://github.com/intelsdi-x/snap#running-snap) with provided configuration file:
-* command line: `snapd -l 1 -t 0 --config cfg.json&`
+* command line: `snapteld -l 1 -t 0 --config cfg.json&`
```
Comment (lmroz) [inline README.md]: In other plugins there is no trailing `&` for this command. Also we prefer to give example using `systemctl` or `service` (see other plugins). Personally I did example in nova collector with downloading example global config to default config location.

File: README.md [hunk 2]
```diff
@@ -85,49 +85,51 @@ Example of running Snap keystone collector and writing data to file.
...
-$ curl -sfLO https://raw.githubusercontent.com/intelsdi-x/snap-plugin-collector-keystone/master/examples/cfg/cfg.json
+$ curl -sfLO https://raw.githubusercontent.com/intelsdi-x/snap-plugin-collector-keystone/master/examples/cfg/cfg.json > /etc/snap/snapteld.conf
 ```
 Ensure to provide your Keystone instance address and credentials.
 
 Ensure [Snap daemon is running](https://github.com/intelsdi-x/snap#running-snap) with provided configuration file:
-* command line: `snapd -l 1 -t 0 --config cfg.json&`
+* initd: `sudo service snap-telemetry start`
+* systemd: `sudo systemctl start snap-telemetry`
+* command line: `sudo snapteld -l 1 -t 0 &`
```
Comment (katarzyna-z) [inline README.md]: Now information how to use snapteld with global configuration (option `--config`) is missed, see line 92

Comment (lmroz) [inline README.md]: Now it's just above in line [88](https://github.com/intelsdi-x/snap-plugin-collector-keystone/pull/12/files/76f345e37487ddfabe7d1ff212a8b21dffcb4e34#diff-04c6e90faac2675aa89e2176d2eec7d8R88 ) Maybe we should link to  [this file](https://github.com/IRCody/snap/blob/31347d829dd3ae57da631904a38f1649972670cb/docs/SNAPTELD_CONFIGURATION.md) in order to clarify how global config works.

Comment (katarzyna-z) [inline README.md]: In my opinion it is good idea to provide link to documentation.

Comment (katarzyna-z) [inline README.md]: In my opinion it is good idea to provide link to documentation.

Comment (mkleina) [inline README.md]: Ok, link provided.

Comment (katarzyna-z) [inline README.md]: It is ok for me

**Pull request merged**